src/services/http.ts

```diff
@@ -5,9 +5,11 @@ export class HTTP {
   logger: Logger;
   apiConfig: ApiConfig;
 
-  constructor(apiConfig: ApiConfig) {
+  static $inject = ['logger'];
+
+  constructor(logger: Logger, apiConfig: ApiConfig) {
```

Comment:
It looks like `apiConfig` isn't injected here.
Are you sure that your app is still working?